docs/site/Inside-Loopback-Application.md

```diff
@@ -0,0 +1,156 @@
+---
+lang: en
+title: 'Inside a Loopback Application'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Inside-Loopback-Application.html
+---
+
+# APIs in a digital era
+
+Digital transformation is changing how data and services are shared across Web/Mobile/IoT platforms.
+APIs allow easy and secure access to digital resources and capabilities of an organization by internal and enternal parties.
+
+Developers are looking out for more and more productive ways of creating and consuming APIs.
+
+Consider a typical `Online Shopping` site. First the UI requirements are gathered. Customers would be provided a home page, a product list, a shopping cart and an order history page.
+
```

Comment:
For a mobile shopping application or web site, the first page usually contains the following information:

- User profile (name, ...)
- Shopping cart icon with status
- Product recommendations
- Order history

I think we can have a screenshot similar as follows:

![image](https://user-images.githubusercontent.com/540892/66853829-07856a80-ef35-11e9-83d3-40ec2250563b.png)

Reply:
agreed